Sprint-1/exercises/count.js

```diff
@@ -2,5 +2,10 @@ let count = 0;
 
 count = count + 1;
 
+/* The current value of the variable count 
+is taken and 1 is added to it. The result 
+is assigned to the variable count
```

Comment:
There is a shorter way to describe statement like this. May I suggest feeding the code to ChatGPT to see how else the code can be described?

Reply:
Without ChatGPT I would assume that it is
`count += 1;`

But just in case I will check if I am right

Hmm, and an interesting result, besides my version, there is another version:
`count++;`

This is basically logical, because Java also uses such an increment construct. But a long time ago I used it in` C/C++`. Thanks CJ for such pleasant nostalgia

Reply:
I was not concerning about the code or your understanding (which you are doing great); I was focusing on "describing code" in English. For example, a more concise description would be "increment count by 1".

Reply:
Thank you, CJ for developing my vocabulary. I am very grateful for that
Tell me, is my phrase "increment construct" bad English? Is it better to say "increment count by N"?